tabpy-server/setup.py

```diff
@@ -25,21 +26,26 @@
               'tabpy_server.psws',
               'tabpy_server.static'],
     package_data={'tabpy_server.static': ['*.*'],
-                  'tabpy_server': ['startup.*', 'state.ini.template']},
+                  'tabpy_server': ['startup.*', 'state.ini']},
     license='MIT',
     install_requires=[
+        'backports_abc',
+        'cloudpickle',
+        'configparser',
+        'decorator',
         'future',
         'futures',
```

Comment:
If bumping to Py3 don't need this anymore, no?

Reply:
You are right, removing `future` and `futures`